src/main/java/com/robin/hoteless/model/Reservation.java

```diff
@@ -0,0 +1,119 @@
+package com.robin.hoteless.model;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    private int id;
+    private Date reservationCreationDate;
+    private int lengthOfStay;
+    private double dailyRate;
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+    private String roomType;
+    private double totalCostOfStay;
+    private static int totalReservations;
+
+    public Reservation(Date reservationCreationDate,
+                       int lengthOfStay,
+                       LocalDate checkInDate,
+                       LocalDate checkOutDate,
+                       double dailyRate,
+                       String roomType,
+                       double totalCostOfStay) {
+        this.id = totalReservations + 10000;
+        this.reservationCreationDate = reservationCreationDate;
+        this.lengthOfStay = lengthOfStay;
+        this.checkInDate = checkInDate;
+        this.checkOutDate = checkOutDate;
+        this.dailyRate = dailyRate;
+        this.roomType = roomType;
+        this.totalCostOfStay = totalCostOfStay;
+        this.totalReservations++;
+    }
+
+    public void show() {
+        System.out.printf(
```

Comment:
Don't have any logic in the model - this needs to be a POJO, no actions inside. Closest to this would be a full toString() method.